### DISCUSSION TITLE
FOLDER STRUCUTRE CHANGED COMPARED TO THE VIDEO?

### DISCUSSION THREAD

**Discussion by adriamrtinez** [chris1111/Wireless-USB-Big-Sur-Adapter#272] [Q&A] [answered]
I ask a question but it was deleted due to a missunderstood!! I'M NOT ASKING ON HOW TO UPGRADE OPENCORE VERSION.

I want to upgrade from Catalina to Big Sur, but ARCHER T3U doesn't work on BigSur. I have seen this could make it work, but file structure (V17) from the assets are different to the video attach on how to install Wireless-USB-BigSur-Adapter. 
I want to know what files form the latest asset "Wireless-USB-Big-Sur-Adapter" need to be executed to install the software or in which folder should I look at?
Thanks...!

**Comment by chris1111** [comment]
Nothings change, Installer is the same I have the Archer T3U working from Catalina to macOS Ventura

**Reply by adriamrtinez** [reply]
But the folder I downloaded contains a READ.me and another folder named "Package sources" with more folders inside

**Comment by chris1111** [comment]
Readme [on Page](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter) download latest release V17

**Comment by chris1111** [comment, marked as answer]
![Screenshot 2023-06-04 at 16 32 17](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/assets/6248794/11f29769-db6b-4b63-85ae-606c8cb7a550)

**Reply by adriamrtinez** [reply]
Thank you. I always downloaded files from the Assets section

**Comment by chris1111** [comment]
Its the same, I create also an assets for the V17 now

**Comment by chris1111** [comment]
make sure you have the good installer because you talk about OpenCore

**Reply by adriamrtinez** [reply]
Okay! Thanks for notifying. I have now download the correct one

**Comment by chris1111** [comment]
You are not in the Wireless.USB.OC.Big.Sur.Adapter page here, read the first page

**Reply by adriamrtinez** [reply]
Sure. Thanks for the time and patience you dedicated to me
